index.php

```diff
@@ -59,13 +59,13 @@
 	Config::load();
 
 	if ($a->max_processes_reached() OR $a->maxload_reached()) {
-		header($_SERVER["SERVER_PROTOCOL"].' 503 Service Temporarily Unavailable');
+		header($_SERVER["SERVER_PROTOCOL"] . ' 503 Service Temporarily Unavailable');
 		header('Retry-After: 120');
-		header('Refresh: 120; url='.App::get_baseurl()."/".$a->query_string);
+		header('Refresh: 120; url=' . App::get_baseurl() . "/" . $a->query_string);
 		die("System is currently unavailable. Please try again later");
 	}
 
-	if (get_config('system','force_ssl') AND ($a->get_scheme() == "http") AND
+	if (get_config('system', 'force_ssl') AND ($a->get_scheme() == "http") AND
 		(intval(get_config('system','ssl_policy')) == SSL_POLICY_FULL) AND
```

Comment:
I don't fully understand your todos.txt system, so I'd rather repeat myself pointing out missing spaces, don't take it personally, at least I know that I reviewed it.